tests/trace_service.py

```diff
@@ -32,7 +32,7 @@ def get_span(self):
 
 
 @pytest.fixture(scope="module")
-def trace_service(request, pytestconfig, logger):
+def trace_service(request, pytestconfig, testdir, logger, certs):
```

Comment:
As was already noted it's one `cert`:
```suggestion
def trace_service(request, pytestconfig, logger, cert):
```

Reply:
OK, Fixed.